### PR TITLE
Jan 8 - Help center, nav, dev tools and overview, sharing, VCCs

### PR DESCRIPTION
**Pull request by rasienko**
@sthielen  For your review, here’s a draft of the first news update of the year. It includes the nav changes and new features you released over the holidays. Let me know if I missed/misunderstood anything or if you have anything else planned for release this week. I tentatively set the date as Friday in case there were some other things we wanted to include. Also, Tim will be updating the VCC screenshots to reflect the new styling, but I expect that will be ready soon. Thanks!

### CODE REVIEW

File: src/asciidoc/docs/develop/testingTemplates.adoc [hunk 1]
```diff
@@ -230,6 +177,10 @@ You can view real-time events related to the sticker context.
 * *Dynamic Receipt* – Debug the receipts that are sent to the buyer and seller, when in-app purchases are implemented with dynamic receipts.
 You can view real-time events related to the receipts.
 
+TIP: To preview a custom context, you can override the environment settings to append the parameter to the URL.
```
Comment (sthielen) [inline src/asciidoc/docs/develop/testingTemplates.adoc:180]: I'm not sure I understand this - is this about showing different contexts inside the debugger? Is this not solved by changing the tabs in the top right?

**Review by sthielen — COMMENTED** [review]
Thanks! This all makes sense to me aside from one piece that I found confusing, but I might not be understanding what it is.

File: src/asciidoc/docs/develop/testingTemplates.adoc (hunk 1 above)
Comment (rasienko) [reply src/asciidoc/docs/develop/testingTemplates.adoc:180]: Thanks! The previous version of the page talked about how to test a custom context (vs. a built-in e.g. sticker) in the live player. Is that still an available test mode, or should we just remove this tip if it's confusing?

**Comment by sthielen** [comment]
I think we can remove it - all of the alternate contexts should be
available in the debugger tabs or the editor live preview buttons

**Comment by rasienko** [comment]
@sthielen I removed that confusing part and also updated some navigation instructions that I missed the first time around. Tim's changes are merged as well, so this PR is ready to go pending your approval. Let me know if we missed anything or need any additional changes. Thanks!

**Review by sthielen — APPROVED** [review]
Looks good to me!

**Pull request merged**